Gruntfile.js

```diff
@@ -210,11 +188,12 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-bumpup');
   grunt.loadNpmTasks('grunt-tagrelease');
   grunt.loadNpmTasks('grunt-saucelabs');
+  grunt.loadNpmTasks('grunt-webpack');
 
-  grunt.registerTask('build', ['replace', 'jshint', 'concat', 'uglify']);
+  grunt.registerTask('build', ['replace', 'jshint', 'uglify:prewebpack', 'webpack', 'uglify:dist']);
```

Comment:
@coryvirok we can replace `uglify:dist` task with the webpack uglify plugin, but seems that the plugin only creates the minified version. Is ok for you?

`uglify:prewebpack` task cannot be removed, we need to minify json2 and tracekit before use them in webpack.